infra.bs

```diff
@@ -547,6 +548,38 @@ ordered sets; implementations can optimize based on the fact that the order is n
 <a for=list>prepend</a> operation otherwise.
 
 
+<h3 id=tuples>Tuples</h3>
+
+<p>A <dfn export>tuple</dfn> is a specification type consisting of a finite ordered
+immutable-sequence of <dfn for=tuple,pair>items</dfn>, each of which has a
+<dfn export for=tuple,pair>name</dfn>.
+
+<p>For notational convenience, a literal syntax can be used to express <a>tuples</a>, by surrounding
+the tuple with ( ) characters and separating the <a for=tuple>items</a> by comma. The
```

Comment:
It seems strange that we'd end up with "`a/b`" for a pair, but `(a, b, c)` for a triple+. I guess that's consistent with existing usage (at least in Fetch), but maybe it's worth changing to `(a, b)` for everything?